presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/Util.java

```diff
@@ -14,8 +14,11 @@
 package com.facebook.presto.sql.planner.iterative.rule;
```

Comment:
Even you copied this commit, I think the commit message which explains what this commit adds is better. Instead, I would just write in PR that the first commit comes from that PR.